src/main/java/org/candlepin/model/ContentCurator.java

```diff
@@ -511,8 +509,7 @@ public Collection<Content> resolveContentsByNamespace(String namespace) {
 
     // Needs an override due to the use of UUID as db identifier.
     @Override
-    @Transactional
-    public void delete(Content entity) {
+        public void delete(Content entity) {
```

Comment:
indention

Reply:
Yup, this needs to be reverted for this PR. It may get removed in the caching PR, but I'll let that PR deal with this change.

Reply:
Fixed